docs/resources/SmbShare.md

```diff
@@ -0,0 +1,115 @@
+
+  storage:
+    pvc:
+      name: datapvc
+      path: dirname
+      spec:
+         ... <k8s pvc spec> ...
+
+  scaling:
+    availabilityMode: clustered
+    minClusterSize: 3
+    groupMode: explicit
+    group: sg1
+
+status:
+  serverGroup: example-share1
+```
+
+## Specification
+
+The desired properties of an SmbShare:
+
+* `shareName`: The name of the share in the SMB protocol (in Samba). Optional.
+  If unspecified the name of the resource will be used.
+* `readOnly`: If set to true clients may only read from the share. Optional.
+  Defaults to false.
+* `browseable`: If set to true clients may see the share name when listing
+  shares on a server. Option. Defaults to true.
+* `securityConfig`: The name of an SmbSecurityConfig resource. The
+  SmbSecurityConfig resource must exist in the same namespace as the SmbShare.
+  Optional. If unspecified the share will default to a simple demo mode
+  security.
+* `commonConfig`: The name of an SmbCommonConfig resource. The SmbCommonConfig
+  resource must exist in the same namespace as the SmbShare. Optional. If
+  unspecified the share will default to simple cluster network access.
+* `storage`: How the share accesses a supporting storage layer
+  * `pvc`: Currently, the samba-operator only supports `PersistentVolumeClaim`s
+    as a supporting storage layer
+    * `name`: The name of a PersistentVolumeClaim to use for SMB shares.
+      Optional. May only be left unset if sibling field `spec` is set.
+    * `path`: The name of a directory within the volume corresponding to
+      the PersistentVolumeClaim. Optional. If unspecified the root of
+      volume will be exposed as a share. Only a single directory name is
+      supported, addititonal subdirectories ("/" characters) are not allowed.
+    * `spec`: Optional. If specified the pvc spec subsection accepts an
+      embedded PersistentVolumeClaim specification. See the [Kubernetes
+      Persistent Volumes Documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+      for details. If specified the PVC will automatically be created and
+      deleted by the samba-operator and thus has a lifecycle paired to the
+      SmbShare.
+* `scaling`: Properties related to resources usage and redundancy
+  * `availabilityMode`: May be either `standard` or `clustered`. Optional.
+    If unspecified defaults to `standard`. Standard availability mode creates
+    one Samba server instance to host the share. Clustered availability mode
+    create one or more CTDB enabled Samba server to host the share in a HA
+    manner. Clustered mode is experimental and needs to be explicitly enabled
+    in the operator.
+  * `minClusterSize`: The minimum number of Samba server "nodes" to host the
+    share. The operator is permitted to run more servers in some conditions.
+  * `groupMode`: My be either `never` or `explicit`. Optional. If unspecified
+    defaults to `never`. An share that is ungrouped (never) is always hosted
```

Comment:
s/An/A

Reply:
Out of curiosity: how did you spot all those typos? do you use a command-line tool or do it manually?

Reply:
done